normandy/recipes/api/serializers.py

```diff
@@ -37,6 +38,25 @@ class Meta:
         ]
 
 
-class BundleSerializer(serializers.Serializer):
+class BundleSerializer(serializers.ModelSerializer):
```

Comment:
d'oh. nope. i originally made `RecipeVersionSerializer` a `Serializer` and then went back to change that and must have changed this instead by mistake.